src/microbe_stage/Microbe.Interior.cs

```diff
@@ -693,10 +693,10 @@ private void HandleCompoundVenting(float delta)
             {
                 amountToVent -= EjectCompound(type, amountToVent, Vector3.Back);
             }
-            else if (Compounds.GetCompoundAmount(type) > 2 * Compounds.Capacity)
+            else if (Compounds.GetCompoundAmount(type) > Compounds.Capacity)
```

Comment:
The multiplier by 2 was here to be kinder to cells that reproduce and don't unnecessarily cause a bunch of compound leakage. I don't think this should have anything to do with infinite compound venting...

Reply:
It put the cell in a state where it had almost twice its capacity of glucose in its storage, which led to `float freeSpace = storage.Capacity - storage.GetCompoundAmount(compound)` in the compound gathering logic returning a negative.

The other change does guard against that, but the cell's stored compounds are still in the wrong state, which is why I added this change. I'll remove this one for now. In Buckly's save, the result is the infinite compound spawning is fixed, but the player cell has to use up 0.5 glucose before any decrease in its stored glucose is visible.